README.md

```diff
@@ -106,6 +106,13 @@ WebAuthn.configure do |config|
   # the User Agent during registration and authentication ceremonies.
   config.origin = "https://auth.example.com"
 
+  # For the case when a relying party has multiple origins (e.g. different subdomains, native client sending android:apk-key-hash:... like origins in clientDataJson, etc...), you can set the allowed origins instead of the single origin above
```

Comment:
What do you think about splitting this into multiple lines like we are doing in the rest of the file? That will make it easier to read as right now you have to scroll horizontally in order to be able to read the whole comment.